hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -0,0 +1,533 @@
+import asyncio
+import math
+import time
+from decimal import Decimal
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
+
+from bidict import bidict
+from bxsolana import Provider
+from bxsolana.provider import WsProvider
+from bxsolana_trader_proto import GetMarketsResponse, api
+from bxsolana_trader_proto.api import (
+    GetAccountBalanceResponse,
+    GetQuotesResponse,
+    GetServerTimeResponse,
+    Market,
+    OrderStatus,
+    Side,
+)
+
+from hummingbot.client.hummingbot_application import HummingbotApplication
+from hummingbot.connector.constants import s_decimal_NaN
+from hummingbot.connector.exchange.bloxroute_openbook import (
+    bloxroute_openbook_constants as CONSTANTS,
```

Comment:
don't import this as all caps

Reply:
Also something other exchanges do

Reply:
weird >:(. I see a mix of this though, and it's very unconventional to import as all caps, so I think best to just not do it all caps

Reply:
Hm okay, the only exchange I see not doing that is `hitbtc` and they created a separate class for constants - I don't have any problem with lowercasing it so I will just do it